website/content/en/preview/concepts/disruption.md

```diff
@@ -177,6 +181,63 @@ To enable interruption handling, configure the `--interruption-queue-name` CLI a
 
 ## Controls
 
+### Disruption Budgets
+
+You can rate limit Karpenter's disruption through the `NodePool.Spec.Disruption.Budgets`. If undefined, Karpenter will default to one budget with `nodes: 10%`. Budgets will take into account nodes that are being deleted for any reason, and will only block Karpenter from disrupting nodes voluntarily through expiration, drift, emptiness, and consolidation.
+
+#### budget.Duration + budget.Schedule
```

Comment:
Should this just be a note or a warning statement rather than a sub-section? I think our depth here is getting pretty deep and I could just see us having the `#### Schedule` and `#### Duration` sections

Reply:
Good callout.